Runtimes/Core/core/CMakeLists.txt

```diff
@@ -269,7 +269,9 @@ if(SwiftCore_ENABLE_VECTOR_TYPES)
     "${CMAKE_CURRENT_BINARY_DIR}/SIMDVectorTypes.swift")
 endif()
 
-set_target_properties(swiftCore PROPERTIES Swift_MODULE_NAME Swift)
+set_target_properties(swiftCore PROPERTIES
+  LINKER_LANGUAGE CXX
+  Swift_MODULE_NAME Swift)
```

Comment:
Are these lines needed for this specific change? Those are the same as #80139, but I don't see we bump the CMake version.

Reply:
Yes, they are related to that change - it is because without this change this library has a circular dependency on itself which then breaks the build.

Reply:
What is the circular dependency? swiftCore should only depend on swiftrt. I don't think swiftrt has any need to depend on swiftCore, no?

Reply:
swiftCore -> swiftCore on Android.